mne/io/ctf/tests/test_ctf.py

```diff
@@ -216,6 +218,42 @@ def test_read_ctf():
                   op.join(ctf_dir, ctf_fname_continuous), 'foo')
 
 
+@testing.requires_testing_data
+def test_rawctf_clean_names():
+    """Test RawCTF _clean_names method"""
+    # read test data
+    raw = read_raw_ctf(op.join(ctf_dir, ctf_fname_catch))
+    raw_cleaned = read_raw_ctf(op.join(ctf_dir, ctf_fname_catch),
+                               clean_names=True)
+    test_channel_names = _clean_names(raw.ch_names)
+    test_info_comps = copy.deepcopy(raw.info['comps'])
+
+    # channel names should not be cleaned by default
+    assert not array_equal(raw.ch_names, test_channel_names)
+
+    chs_ch_names = []
+    for ch in raw.info['chs']:
+        chs_ch_names.append(ch['ch_name'])
```

Comment:
I would have used list comprehension here but it's a nitpick I guess

Reply:
Since this is toward a potential GSoC we can count this as pedagogy not pedantry: please use list comprehension :)